lib/src/__tests__/DatePickerTestingLib.test.tsx

```diff
@@ -1,24 +1,94 @@
-import * as React from 'react';
-import { DatePicker } from '../DatePicker';
+import React, { useState } from 'react';
+import frLocale from 'date-fns/locale/fr';
+import DateFnsUtils from '@date-io/date-fns';
+import enLocale from 'date-fns/locale/en-US';
+import Button from '@material-ui/core/Button';
 import TextField from '@material-ui/core/TextField';
-import { screen } from '@testing-library/react';
-import { createClientRender } from './createClientRender';
+import LocalizationProvider from '../LocalizationProvider';
```

Comment:
In the repository, our imports are inconsistent between using private import path and public ones.
What if every time we have the choice, we go with a public import cc @mui-org/core-team. So in our case

```suggestion
import LocalizationProvider from '@material-ui/pickers/LocalizationProvider';
```

This will encourage us to focus more on end to end.

Reply:
Agree. I try to keep this in mind since it'll help run the test suite for production builds as well among other things. So far it wasn't technically required but for readability it would help a lot.